meetuppizza/settings/base.py

```diff
@@ -29,6 +31,15 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
+MEETUP_KEY = os.getenv("MEETUP_KEY")
+YELP_OAUTH_OBJECT = OAuth1(
+          os.getenv('YELP_CONSUMER_KEY'),
```

Comment:
In Vim, select multiple lines with visual line (Shift+v), then hit `=` to fix indentation. I'm sure there's something similar in Sublime.